.yourbase.yml

```diff
@@ -1,13 +1,12 @@
 dependencies:
   build:
-    - go:1.13.7
+    - go:1.14.1
     - python:3.6
 
 build_targets:
   - name: default
     host_only: true
     commands:
-      - go get -u github.com/yourbase/narwhal
```

Comment:
Fixed our YML and ran `yb-0.0-current run go get -u` followed by `yb-0.0-current run go mod tidy`. Removed import replacements for local versions of Narwhal.